src/main/kotlin/org/openmicroscopy/PublishingPlugin.kt

```diff
@@ -65,19 +65,7 @@ class PublishingPlugin : Plugin<Project> {
         apply<MavenPublishPlugin>()
         apply<ArtifactoryPlugin>()
     }
-
-    // ORIGINAL
-    /* jar {
-        manifest {
-            attributes(
-                    'Build-Timestamp': new java.text.SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ").format(new Date()),
```

Comment:
:+1: for removing commented code. For reference, these are the current values (i.e. they're being set _somewhere_):
```
Manifest-Version: 1.0
Implementation-Title: jar
Implementation-Version: 5.5.6-SNAPSHOT
Built-By: jamoore
Built-Date: 26/03/2020
Class-Path: ...
Built-Gradle: 5.2.1
Built-JDK: 1.8.0_202-ea
```

Reply:
Values are being set, the commented section was more for reference when the plugin was written to be sure that the values were captured

Reply:
values are set with that code
```
    private
    fun Project.configureManifest() {
        plugins.withType<JavaPlugin> {
            tasks.named<Jar>("jar") {
                manifest {
                    attributes["Implementation-Title"] = name.replace(Regex("[^A-Za-z0-9]"), "")
                    attributes["Implementation-Version"] = project.version
                    attributes["Built-By"] = System.getProperty("user.name")
                    attributes["Built-Date"] = SimpleDateFormat("dd/MM/yyyy").format(Date())
                    attributes["Built-JDK"] = System.getProperty("java.version")
                    attributes["Built-Gradle"] = gradle.gradleVersion
                    attributes["Class-Path"] = getRuntimeClasspathConfiguration(project)
                            ?.joinToString(separator = " ") { it.name }
                }
            }
        }
    }
``